scripts/cypress_run.py

```diff
@@ -55,12 +55,7 @@ def get_cypress_cmd(
 
     if use_dashboard:
         # Run using cypress.io service
-        cypress_key = os.getenv("CYPRESS_KEY")
-        command = f"echo {cypress_key} | base64 --decode"
-        cypress_record_key = (
-            subprocess.check_output(command, shell=True).decode("utf-8").strip()
-        )
-        os.environ["CYPRESS_RECORD_KEY"] = cypress_record_key
+        os.environ["CYPRESS_RECORD_KEY"] = os.getenv("CYPRESS_RECORD_KEY", "")
```

Comment:
isn't this kind of a no-op?

Reply:
I had the same thought... let's scrap it!